docs/Change_Log.md

```diff
@@ -3,27 +3,43 @@
 >danger
 >API and Gateway versions below v6 will be discontinued on October 16, 2017, after which they will be non-functioning.
 
-## September 10, 2017 - New Feature
+## New Feature: Channel Categories
+
+#### September 20, 2017
+
+Changes have been made throughout the documentation to reflect the addition of channel categories to Discord. These includes an additional field—`parent_id`—to the base [channel](#DOCS_CHANNEL/channel-obect) object and a new [channel category example] (#DOCS_CHANNEL/channel-object-example-channel-category).
```

Comment:
`[channel category example] (#DOCS_CHANNEL/channel-object-example-channel-category)`  bad space?

Reply:
Thank you kindly